### PR TITLE
chore: linter configuration

### PR DESCRIPTION
**Pull request by michaelavoyan**

### CODE REVIEW

**Review by npyvovartseva — APPROVED** [review]

**Review by ItayPodhajcer — APPROVED** [review]

File: packages/sample-server/test/credentialManifest.test.ts [hunk 1]
```diff
@@ -1,48 +0,0 @@
-"use strict";
```
Comment (nassan) [inline packages/sample-server/test/credentialManifest.test.ts:1]: why are tests being deleted?

File: packages/sample-server/test/initalization.test.ts [hunk 2]
```diff
@@ -1,70 +0,0 @@
-"use strict";
```
Comment (nassan) [inline packages/sample-server/test/initalization.test.ts:1]: same

**Review by nassan — CHANGES_REQUESTED** [review]

File: packages/sample-server/test/initalization.test.ts (hunk 2 above)
Comment (michaelavoyan) [reply packages/sample-server/test/initalization.test.ts:1]: same

File: packages/sample-server/test/credentialManifest.test.ts (hunk 1 above)
Comment (michaelavoyan) [reply packages/sample-server/test/credentialManifest.test.ts:1]: This is not the purpose of the sdk, it's a sample server to which we need to add some simple ui in order to complete the sample for sdk consumer.

**Review by nassan — APPROVED** [review]

**Pull request merged**